File: datasets/IR-Plag/convertai-original-java-rest-python/case-06/plagiarized/L6/09/Level6.py
import sys

def tampil(angka):
    for x in range(9, -1, -1):
        print(angka[x])

def main():
    arrAngka = [0] * 10

    for x in range(len(arrAngka)):
        print("Read a number: ", end="")
        arrAngka[x] = int(input())

    tampil(arrAngka)

if __name__ == "__main__":
    main()

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-06/plagiarized/L6/09/Level6.ts
import * as readline from 'readline';

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

function tampil(angka: number[]): void {
  for (let x = 9; x >= 0; x--) {
    console.log(angka[x]);
  }
}

function main(): void {
  const arrAngka: number[] = new Array(10);

  for (let x = 0; x < arrAngka.length; x++) {
    rl.question('Read a number: ', (input) => {
      arrAngka[x] = parseInt(input);
      if (x === arrAngka.length - 1) {
        tampil(arrAngka);
        rl.close();
      }
    });
  }
}

main();

